README.md

```diff
@@ -62,7 +62,18 @@ source venv/bin/activate
 pip install -r requirements.txt
 ```
 
-### Windows
+### Windows (WSL)
+
+1. Install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and a Linux distribution of your choice (Ubuntu is recommended)
+1. Start bash 
+   1. Install dependencies ``sudo apt-get update; sudo apt-get install python3-pip``
+   1. Install Chrome Browser
+      1. ``wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -``
```

Comment:
Why do you use double ` (backquote) but not single?

Reply:
Ah wasn't aware of this. Just thought this is how it works.

Reply:
Single backquote should do the trick.